### ISSUE TITLE
yabai service fails to start due to accessibility error

### ISSUE THREAD

**Issue by Dalamar42** [koekeishiya/yabai#1730]
When I am trying to start yabai with `yabai --start-service` I am getting repeated prompts to configure accessibility even though I already have.

I have tried the following:
* Grant accessibility as described in the wiki, i.e. manually click + and select yabai from the path shown in `which yabai` --> no fix
* Reset accessibility with `sudo tccutil reset Accessibility` and reconfiguring it after restart --> no fix
* Run `yabai` directly on the terminal --> no fix

The following works:
1. Grant `iterm2` accessibility permissions
2. Run `yabai` on the terminal
3. After a few moments I get a prompt for my password
4. After entering it, it works perfectly, including scripting additions

yabai version: HEAD
MacOs version: 13.3.1

**Comment by koekeishiya** [comment]
You need to codesign the binary when building from source. Instructions are in the wiki.

**Comment by Dalamar42** [comment]
That was it thank you! I did not realise I needed to resign after an update. Solved it by following instructions here: https://github.com/koekeishiya/yabai/wiki/Installing-yabai-(from-HEAD)#updating-to-latest-head
